main.rb

```diff
@@ -1,138 +1,46 @@
-# rubocop:disable Metrics\CyclomaticComplexity, Metrics/MethodLength
-
 require_relative 'create_functions'
 require_relative 'rental'
+require_relative 'library'
 
-class App
-  include CreateFunctions
-
-  def initialize
-    @book_list = []
-    @people = []
-    @rentals = []
-  end
-
-  def list_books
-    if @book_list.empty?
-      puts 'Books list is empty, try adding a new book by selecting number 4'
-      return
-    end
-
-    @book_list.each do |book|
-      puts "Title: #{book.title}, Author: #{book.author}"
-    end
-    puts ''
-  end
-
-  def list_people
-    if @people.empty?
-      puts 'People list is empty, try creating a new person by selecting number 3'
-      return
-    end
-
-    @people.each do |person|
-      puts "[#{person.class}] Name: #{person.name}. ID: #{person.id}, Age: #{person.age}"
+def main
+  app = Library.new
+  option = nil
+  puts 'Welcome To School Library App'
+
+  while option != '7'
+
+    puts 'Please choose an option by entering a number:'
+    puts '1 = List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 = Create a book'
+    puts '5 - Create a rental'
+    puts '6 = List all rentals for a given person id'
+    puts '7 = Exit'
+
+    option = gets.chomp.to_i
+
+    case option
+    when 1
+      app.list_books
+    when 2
+      app.list_people
+    when 3
+      app.people.push(app.create_person)
```

Comment:
A person can not be created when the user wants to create a student. Please resolve the issue.

Reply:
Done ✔